tests/perfclient.py

```diff
@@ -19,12 +19,18 @@ def cli_args(add=lambda x: None, accept_unknown=False):
         help="List of hostnames[,pub_hostnames:ports]. If empty, two nodes are spawned locally",
         action="append",
     )
-    parser.add_argument(
+    client_args_group = parser.add_mutually_exclusive_group()
+    client_args_group.add_argument(
         "-cn",
         "--client-nodes",
         help="List of hostnames for spawning client(s). If empty, one client is spawned locally",
         action="append",
     )
+    client_args_group.add_argument(
+        "--one-client-per-backup",
+        help="If set, allocates one client per backup",
```

Comment:
We already have several ways of specifying clients, and its not clear how they all interact. If we're disallowing clients on remote machines, then `--client-nodes` can be removed/simplified (we want to specify the node a client talks _to_, not the node it runs _on_, but we want to specify as "backup 0" rather than by IP?)

Reply:
I agree about simplifying, but allowing remote clients seems like a good feature to preserve (assuming it does work now, which we don't know because the CI certainly doesn't do that)?

Reply:
I was also thinking of deleting `--client-nodes` too but as @achamayou mentioned, it may be useful to keep this for more interesting perf tests. 

I had to fix a couple of things but remote clients work now.